src/screens/PrivateChat/ChatList/index.js

```diff
@@ -85,6 +88,23 @@ class ChatListScreen extends NavigatorComponent<Props, State> {
       showModal: '',
       loading: false,
     };
+
+    this.props.navigator.setButtons({
```

Comment:
If buttons are not dynamic (i.e. don't change over life cycle of component), you can use `static navigatorButtons = { leftButtons: [], rightButtons: [buttons here] }` instead. It should also make animations between navigation bars more smooth.

Reply:
fixed